arch/x64/mmu.cc

```diff
@@ -15,6 +15,7 @@
 #include <osv/prio.hh>
 #include <osv/elf.hh>
 #include "exceptions.hh"
+#include <algorithm>
```

Comment:
I see in a bunch of files you added this `<algorithm>` but not used it. Is this a mistake?

Reply:
It is used. In this file, it is `std::copy_if()`. 

In general, removing `<*stream>` from `osv/debug.hh` and `<boost/format.hpp>` from `osv/printf.hh` caused some source files like `mmu.cc` no longer compile. These were sometimes symbols from `<algorithm>` but sometimes different standard library headers, which probably were implicitly pulled by `<iostream>` or `<boost/format.hpp>` directly or indirectly.